gcc-cross/Dockerfile

```diff
@@ -4,6 +4,9 @@ MAINTAINER Matt Godbolt <matt@godbolt.org>
 ARG DEBIAN_FRONTEND=noninteractive
 
 RUN mkdir -p /opt mkdir -p /home/gcc-user && useradd gcc-user && chown gcc-user /opt /home/gcc-user
+
+RUN apt-get clean -y && apt-get check -y
```

Comment:
Was this needed? If so, can you explain what it fixes? I'd love to make all the Dockerfiles work roughly the same way and if they're missing this I'd love to add it.

Reply:
asked dbroemmel - answer here: https://github.com/mattgodbolt/compiler-explorer-image/pull/129#issuecomment-435409311